akka-sample-cluster-scala/src/main/scala/sample/cluster/transformation/ScalaPbSerializer.scala

```diff
@@ -0,0 +1,42 @@
+package sample.cluster.transformation
+
+import java.util.concurrent.atomic.AtomicReference
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.BaseSerializer
+import scalapb.GeneratedMessageCompanion
+
+class ScalaPbSerializer(val system: ExtendedActorSystem) extends BaseSerializer {
```

Comment:
Isn't it possible to use the `akka.remote.serialization.ProtobufSerializer` that's included in Akka?
Doesn't the generated ScalaPB classes also have the `parseFrom` method?

Reply:
No @patriknw, it doesn't generate those classes.

Reply:
I think they are there:
```
sbt
console

scala> classOf[sample.cluster.transformation.TransformationMessages.TransformationJob].getDeclaredMethod("parseFrom", Array[Class[_]](classOf[Array[Byte]]):_*)
res0: java.lang.reflect.Method = public static scalapb.GeneratedMessage sample.cluster.transformation.TransformationMessages.TransformationJob.parseFrom(byte[])

```

What happens if you don't configure this serializer and instead have
```
    serialization-bindings {
       "scalapb.GeneratedMessage" = proto
     }
```